src/checkqueue.h

```diff
@@ -143,6 +164,14 @@ class CCheckQueue
         return Loop(true);
     }
 
+    //! Quit execution of any remaining checks.
+    void Quit()
+    {
+       boost::mutex::scoped_lock lock(mutex_fQuit);
+       fQuit = true;
+       LogPrint("parallel_2", "setting fQuit to: %d\n", fQuit);
```

Comment:
these were just intended to be temporary and I've taken them out but keep in mind when you used logprint, they don't actually print out anything unless debug=parallel_2 is set in your bitcoin.conf.  But as i said these are now taken out since the code is fully stable for a while now.